.eslintrc.js

```diff
@@ -1,123 +0,0 @@
-module.exports = {
-    env: {
-        browser: true,
-        es6: true,
-        node: true
-    },
-    extends: [
-        'eslint:recommended',
-        'plugin:jsdoc/recommended',
-        'plugin:json/recommended',
-        'plugin:@typescript-eslint/recommended',
-        'plugin:prettier/recommended',
-        'plugin:promise/recommended',
-        'plugin:import/errors',
-        'plugin:import/warnings',
-        'plugin:import/typescript'
-    ],
-    overrides: [
-        {
-            env: {
-                browser: true,
-                es6: true,
-                node: false
-            },
-            files: ['.js', '.ts'],
-            globals: {
-                $scope: 'writable',
-                cast: 'readonly'
-            }
```

Comment:
I didn't understand the reason for including these items in the `globals` block, and a look at the commit history and a few searches online didn't help.

They have not been carried over to the new config.